app/adapters/moderator.ts

```diff
@@ -1,32 +1,7 @@
-import DS from 'ember-data';
 import OsfAdapter from './osf-adapter';
 
 export default class ModeratorAdapter extends OsfAdapter {
     parentRelationship = 'provider';
-
```

Comment:
We probably need to keep this bunch. Because `moderators` is not first class endpoint, we have to tell the adapter how to figure out the endpoints urls for delete, add, patch.

Reply:
That's what `parentRelationship` is for. See [`osf-adapter.ts`](https://github.com/CenterForOpenScience/ember-osf-web/blob/develop/app/adapters/osf-adapter.ts#L60)

Reply:
From what I understood though, defining `provider` as the `parentRelationship` set that up automatically because it's a thing on `OsfAdapter`?

Reply:
@adlius no. as is riight now, `parentRelationship` is only used in `urlForCreateRecord`.

Reply:
@fabmiz that's true, but DELETE and PATCH will use `links.self`, which should always be there for resources loaded from the api -- `parentRelationship` is most important for building a URL when we don't have one already

Reply:
(might be worth double-checking that the moderators' `links.self` will actually be correct in the API, tho)

Reply:
@aaxelb, Good point.